### DISCUSSION TITLE
REST Application security related recommendations

### DISCUSSION THREAD

**Discussion by kirtimandwade20** [NeoSOFT-Technologies/rest-java-spring-boot#7] [Ideas]
Authentication  & Authorization

**Comment by sasachan** [comment]
@kirtikt 

https://restfulapi.net/security-essentials/

**Comment by Rimasuthar199** [comment]
Here are the some useful recommendation
https://cheatsheetseries.owasp.org/cheatsheets/REST_Security_Cheat_Sheet.html
